keras/wrappers/scikit_learn.py

```diff
@@ -224,7 +224,11 @@ def predict(self, x, **kwargs):
                 Class predictions.
         """
         kwargs = self.filter_sk_params(Sequential.predict_classes, kwargs)
-        classes = self.model.predict_classes(x, **kwargs)
+        # check if the model is sequential or functional
+        if isinstance(self.model, Sequential):
+            classes = self.model.predict_classes(x, **kwargs)
+        else:
+            classes = np.round(self.model.predict(x, **kwargs)).astype(int)
```

Comment:
Rounding predictions to int does not, in fact, predict classes (unless in the specific case of a binary classification problem). Please check the implementation of `predict_classes` on `Sequential`.

Reply:
Thanks for taking time to review the commit! I modified the code again referring to the implementation of "predict_classes" on "Sequential".